README.md

```diff
@@ -35,7 +35,8 @@ If you're already familiar with the conceptual guidelines & the class naming sta
   - [Brace Alignment](#brace-alignment)
   - [Selectors](#selectors)
   - [Properties](#properties)
-  - [Vendor-Prefixed Properties](#vendor-prefixed-properties)
+  - [Vendor-Prefixed Properties](#using-bang-important)
```

Comment:
Huh? 

I should help you set up that MarkdownTOC package for Sublime, so we're using the same settings. It can get a little wonky if things aren't just right.